### ISSUE TITLE
Optimize Upgrade path test E2E run

### ISSUE THREAD

**Issue by mukundansundar** [dapr/cli#1085]
## Describe the proposal
Currently for running all the upgrade paths verification, it takes almost 30 minutes for the test to complete. 

The proposal here is that we only run the latest upgrade path (upgrade and downgrade) against each PR in CLI. 
The full suite of upgrade paths will only be run against the master code in a scheduled run. 

The issue here is, if there are changes in helm version that breaks the previous upgrade paths, that can only be caught after the code has been merged to master. But still that is a small risk to the benefit of having quicker PR check completion. 

Thoughts on this? @artursouza @yaron2 @pravinpushkar @shubham1172 @ItalyPaleAle 

## Release Note

RELEASE NOTE:

**Comment by pravinpushkar** [comment]
I agree with  -  one upgrade + one downgrade test along with ability to trigger full suite with /ok-to-test. This can be used on need basis.

**Comment by shubham1172** [comment]
👋🏻🤖
